openfecwebapp/templates/legal-advisory-opinion-landing.html

```diff
@@ -0,0 +1,53 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal.html' as legal %}
+{% import 'macros/breadcrumbs.html' as breadcrumb %}
+
+{% set breadcrumb_links=[(cms_url + '/legal-resources', 'Legal resources')] %}
+
+{% block title %}Search advisory opinions{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  {{ breadcrumb.breadcrumbs('Advisory opinions', breadcrumb_links) }}
+</header>
+
+<section class="main__content--full data-container__wrapper">
+  {{ legal.filter('Search advisory opinions', 'advisory_opinions') }}
+  <div class="content__section data-container__body">
+    <div class="main">
```

Comment:
@jenniferthibault do we want to use `.main` here? It adds padding above and below the div so it doesn't line up with the filters title:
![image](https://cloud.githubusercontent.com/assets/1696495/17613877/75abf54e-6016-11e6-8bcb-509902ea9129.png)